server/eclipse-project/src/main/java/us/freeandfair/corla/model/CVRContestInfo.java

```diff
@@ -87,7 +87,7 @@
   // this is a list of choice names to make persistence more straightforward; if it
   // were a list of Choice, then the mapping between contests and choices would
   // need to be more complex
-  @ElementCollection(fetch = FetchType.EAGER)
+  @ElementCollection(fetch = FetchType.LAZY)
```

Comment:
Not really. LAZY wherever LAZY works, except for very small collections where the performance benefit of loading them into memory all at once (using one query) outweighs the overhead of that memory usage. 

I'm looking to rework part of the `Persistence` class to make session handling and transaction handling more controlled by us and less by depending on Hibernate's `getCurrentSession()` - I've seen some weirdness where sessions end before they should in the current scheme (which makes LAZY not work in all cases). That will simplify a _lot_ of the transaction setup and such, because it will make `AbstractEndpoint` the sole class that ever begins or ends a transaction, and we'll just say on the various query classes and everything that you need to be in a transaction to invoke them. I also need to examine some of the places where I'm using IDs instead of regular object references (such as `CVRAuditInfo`) to see if lazy loaded object references would make more sense, and do a bit of refactoring in the model classes; there's too much use of `Persistence` creeping into the model classes again. But I digress...